### PR TITLE
Change primary list to be selected by default

### PR DESCRIPTION
**Pull request by ghost**
## Summary

Currently, when adding a user to a list, it's always necessary to select a list.  
But to simplify this, _the primary list_ will be the **default**.

This also avoids the problem of not being able to successfully select the list **twice** if you do the same with pwa.

I'm waiting for your review.  
Thanks

### CODE REVIEW

**Comment by ghost** [comment]
> the problem of not being able to successfully select the list twice if you do the same with pwa.

File: src/client/app/common/views/components/user-menu.vue [hunk 1]
```diff
@@ -77,7 +77,8 @@ export default Vue.extend({
 				select: {
 					items: lists.map(list => ({
 						value: list.id, text: list.name
-					}))
+					})),
+					selected: lists[0].id
```
Comment (acid-chicken) [inline src/client/app/common/views/components/user-menu.vue]: I think that the TypeError is thrown when list is empty.

File: src/client/app/common/views/deck/deck.vue [hunk 2]
```diff
@@ -190,7 +190,8 @@ export default Vue.extend({
 							select: {
 								items: lists.map(list => ({
 									value: list.id, text: list.name
-								}))
+								})),
+								selected: lists[0].id
```
Comment (acid-chicken) [inline src/client/app/common/views/deck/deck.vue]: Same as above.

File: src/client/app/common/views/components/dialog.vue [hunk 3]
```diff
@@ -67,7 +67,8 @@ export default Vue.extend({
 			required: false
 		},
 		select: {
-			required: false
+			required: false,
+			selected: null
```
Comment (syuilo) [inline src/client/app/common/views/components/dialog.vue]: ここには required とか type とかのプロパティしか置けないですね
https://vuejs.org/v2/api/index.html#props

Comment (ghost) [reply src/client/app/common/views/components/dialog.vue]: 修正しました

**Comment by ghost** [comment]
Should I choose a primary list or make the list order changeable?

**Comment by syuilo** [comment]
I have resolved this issue by https://github.com/syuilo/misskey/commit/50251955cdeb5fbb8ff279bbcbea68d2bf606c7f.
Thanks anyway!

**Pull request closed without merging**